Mathlib/CategoryTheory/Comma/Basic.lean

```diff
@@ -327,23 +367,25 @@ def preLeft (F : C ⥤ A) (L : A ⥤ T) (R : B ⥤ T) : Comma (F ⋙ L) R ⥤ Co
       w := by simpa using f.w }
 #align category_theory.comma.pre_left CategoryTheory.Comma.preLeft
 
-instance (F : C ⥤ A) (L : A ⥤ T) (R : B ⥤ T) [F.Faithful] : (preLeft F L R).Faithful where
-  map_injective {X Y} f g h := hom_ext _ _ (F.map_injective (congrArg CommaMorphism.left h))
-    (by apply congrArg CommaMorphism.right h)
+/-- `Comma.preLeft` is a particular case of `Comma.map`,
+but with better definitional properties. -/
+def preLeftIso (F : C ⥤ A) (L : A ⥤ T) (R : B ⥤ T) :
```

Comment:
This is used in constructing instances below so needs to be `reducible`. 

I suspect there are existing violations which are out of scope for this PR.

Reply:
I am sorry I do not understand what you mean.

Reply:
See the other chain of comments for more explanation.